contracts/EpochTokenLocker.sol

```diff
@@ -103,12 +104,16 @@ contract EpochTokenLocker {
         return balanceStates[user][token].pendingWithdraws.stateIndex;
     }
 
+    function getCurrentStateIndex() public view returns(uint32) {
```

Comment:
Not a biggie, but this could have been its own commit.

Reply:
Agreed!